cfme/infrastructure/virtual_machines.py

```diff
@@ -1071,113 +1174,118 @@ class VmAllWithTemplatesDetails(CFMENavigateStep):
     VIEW = InfraVmDetailsView
     prerequisite = NavigateToSibling('All')
 
-    def step(self, *args, **kwargs):
-        sel.click(self.obj.find_quadicon(do_not_navigate=True))
+    def step(self):
+        self.prerequisite_view.sidebar.vmstemplates.tree.click_path('All VMs & Templates')
```

Comment:
Why not go to 'Templates' accordion so that there are fewer entities to search through?